packages/daemon/src/pet-store.js

```diff
@@ -168,9 +194,6 @@ export const makePetStoreMaker = (filePowers, locator) => {
         );
       }
       assertValidId(formulaIdentifier, fromName);
-      if (overwrittenId !== undefined) {
-        assertValidId(overwrittenId, toName);
-      }
```

Comment:
If this id exists we just got it from the memo, and we validate ids before they are memoized.